director/commands/celery.py

```diff
@@ -18,9 +18,9 @@ def beat(dev_mode, beat_args):
     """Start the beat instance"""
     args = [
         "celery",
-        "beat",
         "-A",
         "director._auto:cel",
+        "beat",
```

Comment:
What is the purpose of changing the order of attributes here please?

Reply:
Hello,

Change the order of the attribute is important if you want at terms migrate celery from 4.4.7 to 5.1.2. Without changing the order, celery raised an error related to the -A argument.